### PR TITLE
Add recipe for Pyrosm package

### PR DESCRIPTION
**Pull request by HTenkanen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by HTenkanen** [comment]
@conda-forge/help-python-c `Pyrosm` is ready for review!

File: recipes/pyrosm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  host:
+    - python
+    - pip
+    - cython >=0.28
+    - pyrobuf >=0.9.3
+    - cykhash
+    - python-rapidjson
+    - geopandas
+    - shapely
+    - pygeos
```
Comment (scopatz) [inline recipes/pyrosm/meta.yaml:29]: Do you really need all of these at build time?

Comment (HTenkanen) [reply recipes/pyrosm/meta.yaml:29]: Probably not. Still learning, so don't have yet a good understanding which packages I should include in the `build` and which ones to `run`.  I guess at least cython related packages (`Cython` + `pyrobuf`) should be listed here in the build but maybe I could try to remove the others.

Comment (HTenkanen) [reply recipes/pyrosm/meta.yaml:25]: @scopatz Hmm, well after a couple of try removing things from host produces errors. All those packages mentioned in `host` are hard dependencies for this library.. Hence, I guess I should include all of them? Sorry these might be silly questions, but the [conda docs](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#host) are not very clearly describing what you actually should put here..

Comment (HTenkanen) [reply recipes/pyrosm/meta.yaml:25]: Hmm, well thinking about this and going through the codes makes me quite certain, yes I do need all of those libraries during build time, as those libraries are used from cython files (`.pyx`).

Comment (scopatz) [reply recipes/pyrosm/meta.yaml]: OK Just making sure!

**Pull request merged**